CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 # Changelog
 
-## [Unreleased]
+<!-- ## [Unreleased] -->
 
+## [3.0.1] - 2017-09-12
```

Comment:
`2017-09-13` before 🚢 'ing
Or whichever date we ship this release.

Reply:
Good catch, I’ll update it!